RecoTauTag/HLTProducers/src/CaloTowerCreatorForTauHLT.cc

```diff
@@ -29,9 +29,10 @@ CaloTowerCreatorForTauHLT::CaloTowerCreatorForTauHLT(const ParameterSet& p)
   produces<CaloTowerCollection>();
 }
 
-CaloTowerCreatorForTauHLT::~CaloTowerCreatorForTauHLT() {}
-
 void CaloTowerCreatorForTauHLT::produce(StreamID sid, Event& evt, const EventSetup& stp) const {
+  if (mCone < 0.)
+    return;
```

Comment:
This changes the behaviour of the plugin. If `mCone < 0`, pre-PR the output is an empty collection; post-PR, there is no output collection.

Should be more like
```cpp
  auto cands = std::make_unique<CaloTowerCollection>();
  if (mCone < 0.) {
    evt.put(std::move(cands));
    return;
  }
```

Reply:
Correct, I slipped it: will fix